.circleci/config.yml

```diff
@@ -45,7 +45,7 @@ jobs:
             pip install --progress-bar=off --upgrade pip 'setuptools<49.2.0'
             pip install --progress-bar=off -r test_requirements.txt
             pip install .
-            pip install --progress-bar=off -r doc_requirements.txt
+            pip install --progress-bar=off --no-deps -r doc_requirements.txt
```

Comment:
Doesn't this mean no other packages like docutils will be installed with sphinx?

Reply:
If so, we need two files, one with deps and one without

Reply:
Right, I expect the first shot at this won't work.  Then the missing dependencies can be added to `doc_requirements.txt`, or to a new file that is to be used with `--no-deps` if we need to leave the current `doc_requirements.txt` untouched.

I don't claim this is a great solution.  If anyone has a better approach, I'm happy to close the PR.

Reply:
With sphinx, matplotlib, ipython and pandas (pandas is a doc requirement?), that adds up to a *lot* of dependent packages.  Let's try installing numpy last instead.